README/ReleaseNotes/v634/index.md

```diff
@@ -104,6 +105,11 @@ The following people have contributed to this new version:
 
 * Setting `useHashMapForFind(true)` is not supported for RooArgLists anymore, since hash-assisted finding by name hash can be ambiguous: a RooArgList is allowed to have different elements with the same name. If you want to do fast lookups by name, convert your RooArgList to a RooArgSet.
 
+* function `bindFunction` now supports arbitrary many inputs.
```

Comment:
```suggestion
* The function `RooFit::bindFunction()` now supports arbitrary many input variables when binding a Python function.
```
Don't forget the namespace in the fully qualified name. Also, this only works from PyROOT, at least the way we have implemented so far.